Ginger/GingerCoreCommon/GeneralLib/EncryptionHandler.cs

```diff
@@ -24,19 +24,19 @@
 using Amdocs.Ginger.Common;
 using GingerExternal;
 
-namespace GingerCore
+namespace Amdocs.Ginger.Common
```

Comment:
keep the same namespace 'GingerCore'

Reply:
I have raised New PR now.
Thanks,
Praneeth Reddy